services/server/server.go

```diff
@@ -265,14 +268,35 @@ func LoadPlugins(ctx context.Context, config *srvconfig.Config) ([]*plugin.Regis
 				snapshotters[name] = sn.(snapshots.Snapshotter)
 			}
 
+			shared := true
+			ic.Meta.Exports["policy"] = metadata.PolicyShared
+			if cfg, ok := ic.Config.(*srvconfig.BoltConfig); ok {
+				if cfg.ContentSharingPolicy != "" {
+					if err := cfg.Validate(); err != nil {
+						return nil, err
+					}
+					if cfg.ContentSharingPolicy == metadata.PolicyIsolated {
```

Comment:
`ic.Meta.Exports["policy"]  = cfg.ContentSharingPolicy`. the user can know it is shared or isolated. WDYT?

Reply:
IMHO, I have went through exports of other plugins and didn't think it's related of security. And exporting policy may help operators do debugging.